models/_sector/tokens/avalanche_c/_schema.yml

```diff
@@ -190,3 +190,26 @@ models:
       - name: symbol
       - name: decimals
       - name: collection_name
+
+  - name: tokens_avalanche_c_balances_latest
+    meta:
+      blockchain: avalanche_c
+      sector: tokens
+      contributors: 0xRob
+    config:
+      tags: [ 'tokens','balances', 'avalanche_c' ]
+    description: "Latest token balances"
+    columns:
+      - name: day
+      - name: block_date
+      - name: block_number
+      - name: type
```

Comment:
`token_standard`

Reply:
I think the schema here is copied from the raw balances but it should be the daily.